### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jackgerrits**
## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -1,53 +1,56 @@
...
+      label: Which package was this bug in
+      options:
+        - Core
+        - AgentChat
+        - Extensions
+        - AutoGen Studio
+        - Magentic One
+        - AutoGen Bench
+        - Other
+        - I don't know
```
Comment (ekzhu) [inline .github/ISSUE_TEMPLATE/bug_report.yml:40]: Have we exhausted the choices with the above including "Other"?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -1,53 +1,56 @@
...
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Model used
+      description: If a model was used, please describe it here
```
Comment (ekzhu) [inline .github/ISSUE_TEMPLATE/bug_report.yml:46]:
```suggestion
      description: If a model was used, please describe it here, indicating whether it is a local model or an cloud-hosted model
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -1 +1,5 @@
 blank_issues_enabled: true
+contact_links:
+  - name: Questions or general help
+    url: https://github.com/orgs/community/discussions
```
Comment (ekzhu) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    url: https://github.com/microsoft/autogen/discussions
```

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (jackgerrits) [reply .github/ISSUE_TEMPLATE/bug_report.yml:40]: Think we should remove other "I don't know" for people that aren't sure?

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (jackgerrits) [reply .github/ISSUE_TEMPLATE/config.yml]: Whoops, thank you!

**Review by ekzhu — APPROVED** [review]

**Pull request merged**